packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCPP/ReactPackageProvider.cpp

```diff
@@ -7,22 +7,22 @@
 #include "ReactPackageProvider.g.cpp"
 #endif
 
-#include "CircleViewManagerCPP.h"
-#include "CustomUserControlViewManagerCPP.h"
-#include "SampleModuleCPP.h"
+#include "CircleViewManagerCpp.h"
+#include "CustomUserControlViewManagerCpp.h"
```

Comment:
>CustomUserControlViewManagerCpp [](start = 10, length = 31)

(nit) would also be nice to switch folder names to Pascal case, et SampleLibraryCpp

Reply:
I tried it on my machine, but the changes were not picked up. I may tried it later on my Linux box.

---
In reply to: [359626152](https://github.com/microsoft/react-native-windows/pull/3802#discussion_r359626152) [](ancestors = 359626152)

Reply:
I generally rename the folder to something else, then rename it to the changed case version.  Windows seems to ignore renames that are just case changes.

Reply:
Last time I had success doing it on a Linux box. But it must be done quickly when no other changes are in flight. Otherwise, merging such changes would be quite difficult.

---
In reply to: [359991237](https://github.com/microsoft/react-native-windows/pull/3802#discussion_r359991237) [](ancestors = 359991237)

Reply:
I just learned that case-only changes can break some teams who use rules prohibiting case-only changes. I will do it as Andrew suggested then.

---
In reply to: [360010191](https://github.com/microsoft/react-native-windows/pull/3802#discussion_r360010191) [](ancestors = 360010191,359991237)

Reply:
I wasn't suggesting splitting it into two changes if thats what you are saying.  You should be able to get it in on a single PR.